src/UserSettingsPage.cpp

```diff
@@ -24,9 +24,11 @@
 #include <QString>
```

Comment:
Maybe this file should be renamed to UserSettingsModel to be consistent with our other QML model classes?

Reply:
Well, it is the backend for the UserSettingsPage and also includes other stuff. I am currently also not renaming the chat and welcome page in my other branch, so I think this is fine for now.